debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9ClassLoaderHelper.java

```diff
@@ -82,6 +83,33 @@ public static J9ClassPointer findClass(J9ClassLoaderPointer classLoader, String
 		}
 		return result;
 	}
+	
+	/**
+	 * Find class in a class loader matching search class name.
+	 *
+	 * @param classLoader The class loader used to find the class
+	 * @param searchName The search name of the class
+	 * @throws DDRInteractiveCommandException
+	 * @return J9ClassPointer of a class or null if class is not found.
+	 */
+	public static J9ClassPointer findClassByName(J9ClassLoaderPointer classLoader, String searchName)
+			throws DDRInteractiveCommandException {
+		try {
+			J9ClassPointer clazz = null;
+			Iterator<J9ClassPointer> iterator = ClassIterator.fromJ9Classloader(classLoader);
+			while (iterator.hasNext()) {
```

Comment:
Is this the best way we have to find a class - via a linear search?

Reply:
I wasn't able to find an existent search method to search classes in a class loader based on `String className`, not too sure how else I should implement this